src/elody/client.py

```diff
@@ -95,3 +95,25 @@ def upload_file_from_url(self, entity_id, filename, file_url, identifiers=None):
             upload_location, files={"file": mediafile}, headers=self.headers
         )
         return self.__handle_response(response, "Failed to upload mediafile")
+
+    def add_object_metadata(self, collection, identifier, payload):
+        if collection == "entities":
+            url = f"{self.elody_collection_url}/{collection}/{identifier}/metadata"
+            payload = payload if isinstance(payload, list) else [payload]
+            response = requests.patch(url, json=payload, headers=self.headers)
+            if (response.status_code == 400) and (
+                response.json()["message"].endswith("has no metadata")
```

Comment:
The parentheses are superfluous. Can you remove them and format the file with Black?